tools/lint-svg

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env node
+const XMLDoc = require("xmldoc").XmlDocument;
+const chalk = require("chalk");
+const util = require("util");
+util.inspect.styles.string = "cyan";
+util.inspect.styles.regexp = "grey";
+const inspect = value => util.inspect(value, {colors: true});
+
+/**
+ * All the rules we use for SVG linting
+ * Are given a single argument, the parsed SVG object
+ * Should throw an error if it fails
+ */
+const RULES = [
+    function rootAttributes(svg) {
+        const expected = {
+            xmlns: "http://www.w3.org/2000/svg", 
+            viewBox: "0 0 24 24", 
+            "aria-labelledby": "title", 
+            role: "img"
+        };
+        const msgs = [];
+
+        for (let attr in expected) {
+            if (svg.attr[attr] !== expected[attr]) {
+                msgs.push(`SVG attribute ${chalk.bold(attr)} should be ${inspect(expected[attr])}, was ${inspect(svg.attr[attr])}`);
```

Comment:
```
SVG attribute aria-labelledby should be 'title', was undefined
SVG attribute role should be 'img', was undefined
```

If `aria-labelledby` or `role` do not equal `title` or `img`, those attributes likely do not exist in the file at all. Therefore, I think that these messages would be more clear if they were worded like this:

```
SVG attribute aria-labelledby is undefined; should be 'title'
SVG attribute role is undefined; should be 'img'
```

Note that I am also using present tense (`is`) instead of past tense (`was`). That's just me nitpicking.